server.js

```diff
@@ -75,4 +76,18 @@ async function handlePostBooks(req, res) {
   }
 }
 
+async function handleDeleteBooks(req, res){
+  // res.status('this working in delete');
+  // console.log(req.params.id);
+  // let { id } = req.params
+  // res.send('route hit');
+  let id = req.params.id;
+  try {
+    await Book.findByIdAndDelete(id);
+    res.status(204).send('cat deleted');
```

Comment:
- Good work on delete and create! 
- Our message here is for cats instead of books so we can update it 👍